src/main/java/com/strangeone101/holoitemsapi/enchantment/EnchantListener.java

```diff
@@ -101,183 +105,211 @@ public void onProjectileLaunch(ProjectileLaunchEvent event) {
     }
 
     /**
-     * Handles anvil craftings regarding custom enchantments and custom items.
-     * @param event
+     * Makes sure that after a player closes an anvil inventory, their instant build
+     * ability gets disabled.
      */
-    @EventHandler(ignoreCancelled = true)
-    public void onPrepareAnvil(PrepareAnvilEvent event) {
+    @EventHandler(priority = EventPriority.MONITOR)
+    public void onInventoryClose(InventoryCloseEvent event) {
+        if (!(event.getInventory() instanceof AnvilInventory) ||
+                !(event.getPlayer() instanceof Player player) ||
+                player.getGameMode() == GameMode.CREATIVE)
+            return;
 
-        var player = event.getView().getPlayer();
+        new PlayerAbilitiesPacket(player, false).sendPacket(player);
+    }
 
-        // Avoid creative bypasses
-        if (player.getGameMode() == GameMode.CREATIVE)
+    /**
+     * Handles anvil craftings regarding custom enchantments and custom items.
+     */
+    public void onPrepareAnvil(PrepareAnvilEvent event) {
```

Comment:
```suggestion
    @EventHandler()
    public void onPrepareAnvil(PrepareAnvilEvent event) {
```
Don't forget to readd event handler

Reply:
Oh... Haha whoops. I'd meant to only remove the ignoreCanceled since it's not a cancellable event. My bad.